src/odemis/acq/align/delphi.py

```diff
@@ -321,7 +323,7 @@ def _DoDelphiCalibration(future, main_data):
             future.resolution_shiftf = ResolutionShiftFactor(main_data.bsd,
                                                              main_data.ebeam, sem_stage,
                                                              main_data.ebeam_focus,
-                                                             good_focus)
```

Comment:
Please explain why using the hole focus. Is it looking again at a hole at this step?